torch/quantization/QConfig.py

```diff
@@ -63,3 +65,25 @@ def __new__(cls, weight):
 
 default_qat_qconfig = QConfig(activation=default_fake_quant,
                               weight=default_weight_fake_quant)
+
+def get_default_qconfig(backend='fbgemm'):
+    if backend == 'fbgemm':
+        qconfig = QConfig(activation=HistogramObserver.with_args(reduce_range=True),
+                          weight=default_per_channel_weight_observer)
```

Comment:
so we do per-channel only for fbgemm, not for mobile?

Reply:
Mobile does not support per-channel kernels currently. If and when QNNPACK supports it, we can change that.